rca/enquire_to_study/birdbath.py

```diff
@@ -0,0 +1,40 @@
+from faker import Faker
+
+from rca.enquire_to_study.models import EnquiryFormSubmission
+from rca.utils.birthbath import BaseModelAnonymiser
+
+fake = Faker()
+
+
+class EnquiryFormSubmissionAnonymiser(BaseModelAnonymiser):
+    """
+    Anonymise EnquiryFormSubmission
+    """
+
+    model = EnquiryFormSubmission
+
+    anonymise_fields = [
+        "first_name",
+        "last_name",
+        "email",
+        "phone_number",
+        "country_of_residence",
+        "country_of_citizenship",
+        "city",
+    ]
+
+    def get_anonymous_field_value(self, field_name, obj):
+        if field_name == "first_name":
+            return fake.first_name()
+        if field_name == "last_name":
+            return fake.last_name()
+        if field_name == "email":
+            return fake.ascii_email()
```

Comment:
I was rather taken aback by the fake names and emails as they look like real ones! But after testing, I can confirm they are fake. Perhaps a less realistic fake credentials would be better. Perhaps appending `.fake` to the end of emails? E.g. `return fake.ascii_email() + .fake`. Also to avoid accidentally sending mails to these addresses as they can have real destinations.

Reply:
@victoriachan This is a valid point. Having looked at the data here `EnquiryFormSubmission` I don't think there's actually any reason to keep it around and we should process them by deleting them.

What do you think about that?

Reply:
OK, yea. Good idea. I suppose we normally just delete them, like we do the other form submission.

Reply:
I've replaced this processor with the ModelDeleter